cmake/env/pol_global_config.h.in

```diff
@@ -9,11 +9,10 @@
 #cmakedefine HAVE_MYSQL
 #cmakedefine ENABLE_BENCHMARK
 
-#define POL_VERSION @POL_VERSION@
 #define POL_VERSION_STR "@POL_VERSION_STR@"
 #define POL_VERSION_NAME "@POL_VERSION_NAME@"
-#define POL_VERSION_ID "POL@POL_VERSION_STR@ @POL_VERSION_NAME@"
+#define POL_VERSION_ID "@POL_VERSION_STR@ @POL_VERSION_NAME@"
```

Comment:
Looks fine to me. But do you know if any scripts expects "POL" in front of the version? I guess we could fine-tune this later. It might be good to start with something non-numeric though.

Also, somewhere along the way we lost the hyphen in the version string:
```suggestion
#define POL_VERSION_ID "@POL_VERSION_STR@ - @POL_VERSION_NAME@"
```
which the shard status page was relying on to split the version number from the version name.